tests/test_find_xml_file.py

```diff
@@ -0,0 +1,10 @@
+from find_xml_file.find_xml_file import find_xml_file
+
+
+def test_no_files_returns_None():
```

Comment:
Are we testing for a specific file? It might be helpful to be more descriptive here. "test_no_xml_files_returns_None():"